request/values_parser.go

```diff
@@ -192,6 +192,22 @@ func (v *Values) Time(reference string, layout string) *time.Time {
 	return &timeValue
 }
 
+// the api side is always a string, handle this the same as Time
+func (v *Values) MultiTime(reference string, layout string) *time.Time {
```

Comment:
You can simply call `v.Time(reference, layout)` instead of copying the implementation.